code/_core/mob/living/advanced/player/_player.dm

```diff
@@ -221,12 +229,12 @@
 
 	return TRUE
 
-/mob/living/advanced/player/proc/prestige(var/skill_id)
+/mob/living/advanced/player/proc/prestige(skill_id)
 	if(!prestige_count[skill_id])
 		prestige_count[skill_id] = 1
 	else
 		prestige_count[skill_id] += 1
-	set_skill_level(skill_id,5)
+	set_skill_level(skill_id, (prestige_count[skill_id]*5))
```

Comment:
I'm iffy about this PR because theoretically if you prestige 20 times you can then prestige an unlimited amount of times.

Reply:
Actually nevermind, I understand.